### PR TITLE
feat(onboarding): Split Experiments into its own flow

### PR DESCRIPTION
**Pull request by danielbachhuber**
## Problem

**Feature Flags & Experiments** currently have a combined onboarding flow. It would provide a better onboarding experience if they were split into two.

## Changes

Splits **Feature Flags & Experiments** into **Feature Flags** and **Experiments** on `/products`:

CleanShot 2024-11-12 at 06 25 37@2x

When you click on "Experiments", you'll enter a three step onboarding flow (SDK install, Proxy, Invite teammates):

CleanShot 2024-11-08 at 08 08 00@2x

This PR integrates nicely with https://github.com/PostHog/posthog/pull/26018:

CleanShot 2024-11-12 at 06 26 20@2x

It's currently missing at least these SDKs (compared to Feature Flags):
* ~Android~ added
* ~iOS~ added
* ~Next.js~ added
* ~Nuxt.js~ added
* ~React~ added
* ~Svelte~ added

~We don't have the code snippets for them in Experiments. I might add them in this PR, or a follow-up PR.~ added

## How did you test this code?

Clicked through the entire flow.

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@jurajmajerik Do you have much of an opinion on what should go into a v1 of an Experiments onboarding flow? We can discuss sync tomorrow too, instead.

**Comment by jurajmajerik** [comment]
Two things would be nice:
- Make sure SDK/Feature flags are initialized correctly (can reuse the same screen as Feature Flags)
- Some sort of product intro - I _think_ it can be synced with what we have on the website? (https://posthog.com/experiments) - check with @raquelmsmith

**Comment by danielbachhuber** [comment]
> * Some sort of product intro - I _think_ it can be synced with what we have on the website? ([posthog.com/experiments](https://posthog.com/experiments))

@jurajmajerik Raquel and I discussed today. I'd like to do this as a follow-up experiment so that we can see if it improves upon the baseline of whatever this establishes.

**Review by raquelmsmith — APPROVED** [review]
Nice work!!

File: frontend/__snapshots__/scenes-other-products--products--light.png
[the remark this answers is not in the document]
Comment (danielbachhuber) [reply frontend/__snapshots__/scenes-other-products--products--light.png:1]: I agree, updated with 70b98c91e61a2a23acdcf73376d0dad25c8b12c3

**Pull request merged**